### ISSUE TITLE
Mercury exposure

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11306]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 40287162 Pink disease was once a widespread phenomenon, known to physicians throughout the Western world. Its prevalence declined massively once the source, mercury, was identified in several products. In modern medicine, pink disease rarely forms a differential and is known only to few physicians, despite the long-term effects of this illness. This report details a case of a man who suffered from pink disease as a child following his mother applying teething powder containing inorganic mercury to his gums for 12 weeks, from the age of 6 months. He received no treatment at the time. Throughout his lifetime, however, the man (in his 70s) has been diagnosed as infertile, and with bronchiectasis, despite a lack of convincing history to suggest an aetiology. The cause of these pathologies remains elusive, either organically occurring or components of Young's syndrome. 

PMID: 30969689 Young syndrome is a rare inherited syndrome. It has correlations with mercury exposure. Since the implementation of restrictions on mercury use, the incidence of Young syndrome has also declined.

**Parent term (use hpo.jax.org/app)**
Exposure history HP:4000110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Infantile acrodynia ORPHA:247165 (Pink disease)
 Young syndrome ORPHA:3471

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001414
- label: Mercury exposure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:40287162
- def: History of exposure to mercury.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11306
- synonyms:
